.gitignore

```diff
@@ -66,6 +69,7 @@ dependency-reduced-pom.xml
 !.settings/org.eclipse.m2e.core.prefs
 !src/main/resources/com/datalogics/pdf/hsm/samples/pdfjavatoolkit-ds.pdf
 !user-documentation.pdf
+!src/test/resources/com/datalogics/pdf/hsm/samples/util/test.properties
```

Comment:
Actually, is ignoring all `.properties` files the smartest way to do this? We want to keep out the password file, but users might add other `.properties` files of their own that they want to keep in version control.